websites/M/Mugzone/dist/metadata.json

```diff
@@ -8,7 +8,7 @@
     "category": "games",
     "logo": "https://i.imgur.com/HP5fcgM.png",
     "thumbnail": "https://i.imgur.com/JkhV3Q0.png",
-    "version": "1.0.1",
+    "version": "1.1.1",
```

Comment:
Shouldn't it go to 1.1.0?

Reply:
It should be 1.0.2

Reply:
I edited the version number to 1.1.0.Since on PreMID discord they said adding 0.1.0 is for minor changes and 0.0.1 for bug fixes

Reply:
What xd minor changes are third number not second.